packages/models-library/src/models_library/services.py

```diff
@@ -1,30 +1,30 @@
-import sys
+"""
+
+NOTE: to dump json-schema from CLI use
+    python -c "from models_library.services import ServiceDockerData as cls; print(cls.schema_json(indent=2))" > services-schema.json
+"""
 from enum import Enum
-from pathlib import Path
 from typing import Dict, List, Optional, Union
 
 from pydantic import BaseModel, EmailStr, Extra, Field, HttpUrl, constr, validator
 from pydantic.types import PositiveInt
 
-from .constants import VERSION_RE
+from .basic_regex import VERSION_RE
 
-current_file = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve()
 
+SERVICE_KEY_RE = r"^(simcore)/(services)/(comp|dynamic|frontend)(/[^\s/]+)+$"
+KEY_RE = SERVICE_KEY_RE  # TODO: deprecate this global constant by SERVICE_KEY_RE
 
-KEY_RE = r"^(simcore)/(services)/(comp|dynamic|frontend)(/[^\s/]+)+$"
 PROPERTY_TYPE_RE = r"^(number|integer|boolean|string|data:([^/\s,]+/[^/\s,]+|\[[^/\s,]+/[^/\s,]+(,[^/\s]+/[^/,\s]+)*\]))$"
+PROPERTY_KEY_RE = r"^[-_a-zA-Z0-9]+$" # TODO: should be a UUID_RE instead??
```

Comment:
I don't think this is a UUID, is that not the input/output names? then they can be almost anything...

Reply:
you want a uuid? we did not have a uuid and I want as well a uuid ... but i am afraid to add right now a uuid pattern there (which is ready in basic_regex module)

Reply:
no don't put a uuid here. this will break a lot of things if you do. these keys are not supposed to be uuids but the name of the ports